.github/workflows/scorecards.yaml

```diff
@@ -45,12 +45,12 @@ jobs:
 
       # Upload the results as artifacts (optional). Commenting out will disable uploads of run results in SARIF
       # format to the repository Actions tab.
-      - name: "Upload artifact"
-        uses: actions/upload-artifact@97a0fba1372883ab732affbe8f94b823f91727db # v3.pre.node20
-        with:
-          name: SARIF file
-          path: results.sarif
-          retention-days: 5
+      # - name: "Upload artifact"
```

Comment:
Do you want to add TODO with a bug number here?

Reply:
I think we should revert this ASAP. I don't think the TODO is really necessary